.ci/shippable.sh

```diff
@@ -131,6 +131,15 @@ validate-ci-temp-empty)
   exit $fail
   ;;
 
+git-status)
+  if [ $(git status | grep "Untracked files" | wc -l) -gt 0 ]; then
```

Comment:
No need to grep here. It is not a stable approach.
Please use `git status --porcelain=v1 --untracked-files=normal` instead. Here and in all such cases.

Reply:
I agree.  Fixed.